core/container/small_array/small_array.odin

```diff
@@ -102,6 +116,19 @@ consume :: proc "odin" (a: ^$A/Small_Array($N, $T), count: int, loc := #caller_l
 	a.len -= count
 }
 
+ordered_remove :: proc "odin" (a: ^$A/Small_Array($N, $T), index: int, loc := #caller_location) {
```

Comment:
could maybe do what the builtin `order_remove` does -> https://github.com/odin-lang/Odin/blob/master/core/runtime/core_builtin.odin#L61

maybe also use runtime.bounds_check_error_loc with `#no_bounds_check` flag